pkg/providers/instancetype/instancetype.go

```diff
@@ -225,7 +225,7 @@ func (p *Provider) GetInstanceTypes(ctx context.Context) ([]*ec2.InstanceTypeInf
 	}
 	if p.cm.HasChanged("instance-types", instanceTypes) {
 		logging.FromContext(ctx).With(
-			"instance-type-count", len(instanceTypes)).Debugf("discovered EC2 instance types")
+			"instance-type-count", len(instanceTypes)).Debugf("discovered instance types")
```

Comment:
```suggestion
			"count", len(instanceTypes)).Debugf("discovered instance types")
```

This is already scoped to instance types, so maybe just "count"?